server/src/domain/library/library.service.ts

```diff
@@ -68,8 +71,16 @@ export class LibraryService extends EventEmitter {
 
   async init() {
     const config = await this.configCore.getConfig();
+
     const { watch, scan } = config.library;
-    this.watchLibraries = watch.enabled;
+
+    if (watch.enabled) {
+      // This ensures that library watching only occurs in one microservice
+      // TODO: we could make the lock be per-library instead of global
```

Comment:
I personally don't think we ever need this.

Reply:
I agree as well

Reply:
You are correct, this would make it impossible to switch to watching at runtime